Sec 3 edits for Dec 2020 (#176)
hassanobeid1994/tr_b_causal_2020: article/sections/_3_graph_overview.tex

Figure \ref{fig:simple-graph-confounded} shows an indirect connection between Z and Y that goes through U.
Therefore, even if we condition on X, omitting U will yield biased results of the relationship between Z and Y, due to the variation in both Z and Y caused by U.
The structural equations also show that the ignorability assumption of Z does not hold if we only control for X, and we risk obtaining biased estimates of the causal effect of Z on Y if we fail to account for U, sometimes referred to as a common cause.

> While the set of structural equations and the DAGs illustrate the same points in theory, one may rightfully argue that as the complexity of the problem increases, with many covariates available for the modeler to choose from, a graphical representation of the relationship between those covariates quickly becomes more intuitive and easier to follow. 

Author: I think the sentence should be split so that it doesn't express many thoughts at once. Here's an example:
```suggestion
While the set of structural equations and the DAGs illustrate the same points in theory, the clarity of structural equations scales poorly with a problem's complexity. 
As the number of covariates in one's causal system increases, a graphical representation of the relationship between those covariates quickly becomes more intuitive and easier to follow.
```

> More importantly, a graphical illustration of the modeler's assumptions reduces the toll of following those assumptions on the reader, which is key to avoiding the misuse of those models in real-world applications.

Author: Again, I'd suggest splitting the sentence in two.  Additionally, I suggest making the first sentence use more active voice. Here's an example.
```suggestion
More importantly, graphically illustrating our assumptions eases the communication of these assumptions to the reader.
This communication is key to avoiding the misuse of our models in real-world applications.
```

> This advantage of DAGs is of particular relevance in travel demand modeling applications, where the number of variables included in a typical model is large and highly intertwined. 

Author: Last one 🙂, a sentence splitting suggestion and a suggestion to flip the order of your sentences to add more connection to the sentence on the previous line. Here's an example to show what I mean.:
 ```suggestion
 For example, consider travel demand modeling applications.
 Here, the communication advantage of DAGs is of particular relevance, as travel demand models typically involve many variables with complex interdependencies.
```
